helper/config/decode.go

```diff
@@ -145,3 +146,26 @@ func uint8ToStringHook(f reflect.Kind, t reflect.Kind, v interface{}) (interface
 
 	return v, nil
 }
+
+func updateString(oldString, newString string) string {
+	if oldString != "" {
+		if newString != "" {
+			return newString
+		}
+		return oldString
+	}
+	return newString
+}
+
+func updateMap(oldMap, newMap map[string]string) map[string]string {
+	if oldMap == nil {
+		oldMap = newMap
+	} else {
+		for newk, newv := range newMap {
+			if newk != "" {
```

Comment:
will this happen ❓

Reply:
hm. I don't think so but I'll run a test or three before removing it.